mycore-oai/src/main/resources/META-INF/resources/oai/oai2.xsl

```diff
@@ -339,20 +349,17 @@ p.intro {
 -->
 
 <xsl:template match="ri:rightsManifest" xmlns:ri="http://www.openarchives.org/OAI/2.0/rights/">
-  <h2>Rights Manifest</h2>
   <xsl:variable name="appliesTo" select="substring-after(@appliesTo,'#')" />
+  <h2>Rights Manifest for <xsl:value-of select="$appliesTo" /></h2>
   <xsl:for-each select="ri:rights">
-    <h2 class="oaiRecordTitle">Applies to: <xsl:value-of select="$appliesTo"/></h2>
-    <div class="oaiRecord">
-      <xsl:if test="ri:rightsReference">
-        <h3>Reference</h3>
-        <xsl:variable name="rightsReference" select="ri:rightsReference/@ref" />
-        <div><a href="{$rightsReference}"><xsl:value-of select="$rightsReference" /></a></div>
-      </xsl:if>
-      <xsl:if test="ri:rightsDefinition">
-        <xsl:apply-templates select="ri:rightsDefinition/*" />
-      </xsl:if>
-    </div>
+    <xsl:if test="ri:rightsReference">
+      <h3>Reference</h3>
+      <xsl:variable name="rightsReference" select="ri:rightsReference/@ref" />
+      <div><a href="{$rightsReference}"><xsl:value-of select="$rightsReference" /></a></div>
+    </xsl:if>
+    <xsl:if test="ri:rightsDefinition">
+      <xsl:apply-templates select="ri:rightsDefinition/*" />
+    </xsl:if>
```

Comment:
Does not seems to be related to the ticket and removes <div> and @class. Why?

Reply:
As mentioned in the JIRA ticket and the initial post in this PR, I've also harmonized the [recently added display of Rights Definition descriptions](https://github.com/MyCoRe-Org/mycore/pull/1694) (something that nobody should rely on as of yet) with the newly added support for Dublin Core descriptions.

This could _technically_ be two separate PRs, but that seemed to be overly fine grained for me.